src/modules/pos/validator/store/actions/staking.js

```diff
@@ -108,10 +108,21 @@ export const stakesRetrieved = () => async (dispatch, getState) => {
   const { network, account } = getState();
   const address = account.current?.metadata?.address;
   try {
-    const votes = await getVotes({ network, params: { address } });
+    const stakes = await getStakes({ network, params: { address } });
+    const validators = await getValidatorList({
+      addresses: stakes.data.map(({ delegateAddress }) => delegateAddress),
```

Comment:
The terminology has changed
```suggestion
      addresses: stakes.data.map(({ address }) => address),
```